llama-index-integrations/readers/llama-index-readers-s3/llama_index/readers/s3/base.py

```diff
@@ -106,11 +106,16 @@ def _get_simple_directory_reader(self) -> SimpleDirectoryReader:
             fs=s3fs,
         )
 
-    def load_s3_files_as_docs(self, temp_dir=None) -> List[Document]:
+    def _load_s3_files_as_docs(self) -> List[Document]:
         """Load file(s) from S3."""
         loader = self._get_simple_directory_reader()
         return loader.load_data()
 
+    def _aload_s3_files_as_docs(self) -> List[Document]:
+        """Asynchronously load file(s) from S3."""
+        loader = self._get_simple_directory_reader()
+        return loader.aload_data()
```

Comment:
This should have an await right?

Reply:
Good catch, fixed https://github.com/run-llama/llama_index/pull/14319/commits/8526db6c9d749df83ed4edd5ee68f9fc728991d2